doc/configuration_options.rst

```diff
@@ -431,6 +431,9 @@ The following options are valid in version ``1`` (beside the generic options):
 
 * ``jenkins_job_timeout``: the job timeout for *CI* jobs.
 
+* ``jenkins_job_upstream_trigger``: name(s) of other CI job(s) which, when
```

Comment:
The singular / plural is a bit confusing. Since the configuration option must be a list it should be only plural.

Reply:
Done.

Reply:
> Done.

Still looks the same?

Reply:
Looks plural to me: `jenkins_job_upstream_triggers`

Reply:
If I would have seen the incremental diff I would have noticed that you appended the `s` to the config name. I was referring to the description text. It should not mix both singular and plural but only be plural.

Reply:
Changed.